.github/workflows/rcTagAndRelease.yml

```diff
@@ -1,6 +1,7 @@
 name: rc tag/release
 
 on:
+  workflow_dispatch:
```

Comment:
Just to be able to manually kick this off if needed.
Allows you to create both the tag and release with one click instead of mucking about with both git and the Github UI (or gh cli) for it.